src/DependencyInjection/SonataDoctrineORMAdminExtension.php

```diff
@@ -38,6 +38,7 @@ public function load(array $configs, ContainerBuilder $container): void
         $loader->load('doctrine_orm_filter_types.xml');
 
         $bundles = $container->getParameter('kernel.bundles');
+        \assert(\is_array($bundles));
```

Comment:
seems strange having to add this around each getParameter to ensure type

Reply:
getParameter is considered as returning `mixed` for phpstan because of the symfony phpdoc